### PR TITLE
Example code for MuLawDecoding  Transform

### PR DESCRIPTION
**Pull request by harishsdev**
Example code for MuLawDecoding  Transform

### CODE REVIEW

**Comment by harishsdev** [comment]
@nateanl please he to review this PR

File: torchaudio/transforms.py [hunk 1]
```diff
@@ -699,6 +699,10 @@ class MuLawDecoding(torch.nn.Module):
 
     Args:
         quantization_channels (int, optional): Number of channels. (Default: ``256``)
+    Example
```
Comment (nateanl) [inline torchaudio/transforms.py:702]: Let's add a line before ``Example``
```suggestion

    Example
```

**Review by nateanl — CHANGES_REQUESTED** [review]
LGTM. Just need to fix the nit.

**Comment by nateanl** [comment]
Could you also clean the commit history in the PR? You can do it by rebase to the main branch and force push it.

**Review by nateanl — APPROVED** [review]
LGTM. Next time when you create the PR, could you reset your local branch to the latest main branch? So that we can have only the related commits in the commit history. Thanks! @harishsdev

**Comment by harishsdev** [comment]
git fetch origin
git reset --hard origin/master

above commands are correct,i will reset from next onwards

**Pull request merged**